{"2d0f4a8b4fc3ddfad9408b2e5289fdd6":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"48b5869a5b4f6684","status":"failed","statusDetails":"Setup failed:\nWebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Error executing adbExec. Original error: 'Command '/usr/local/lib/android/sdk/platform-tools/adb -P 5037 -s emulator-5554 shell 'settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy'' exited with code 1'; Command output: adb: device offline\n\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Error executing adbExec. Original error: 'Command '/usr/local/lib/android/sdk/platform-tools/adb -P 5037 -s emulator-5554 shell 'settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy'' exited with code 1'; Command output: adb: device offline\n\n    at getResponseForW3CError (/opt/hostedtoolcache/node/22.17.0/x64/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/errors.js:1143:9)\n    at asyncHandler (/opt/hostedtoolcache/node/22.17.0/x64/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/protocol.js:487:57)\n\nAlso teardown failed:\nSeveral failures occurred:\n\n1) CalledProcessError: Command '['adb', 'pull', '/data/local/tmp/Beta_1_-_Test_Enable_wifi_settings.mp4', '/home/runner/work/test-uiautomator2-driver/test-uiautomator2-driver/TestRecord/Beta_1_-_Test_Enable_wifi_settings.mp4']' returned non-zero exit status 1.\n\n2) File '/home/runner/work/test-uiautomator2-driver/test-uiautomator2-driver/TestRecord/Beta_1_-_Test_Enable_wifi_settings.mp4' does not exist.\n\n3) FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/test-uiautomator2-driver/test-uiautomator2-driver/TestRecord//Beta_1_-_Test_Enable_wifi_settings.mp4'","time":{"start":1750911957742,"stop":1750911968323,"duration":10581}}]},"e88c0dba020b1232e86f3386a0e31fd9":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"c3d205125ff43113","status":"failed","statusDetails":"Setup failed:\nWebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Error executing adbExec. Original error: 'Command '/usr/local/lib/android/sdk/platform-tools/adb -P 5037 -s emulator-5554 shell 'settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy'' exited with code 1'; Command output: adb: device offline\n\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Error executing adbExec. Original error: 'Command '/usr/local/lib/android/sdk/platform-tools/adb -P 5037 -s emulator-5554 shell 'settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy'' exited with code 1'; Command output: adb: device offline\n\n    at getResponseForW3CError (/opt/hostedtoolcache/node/22.17.0/x64/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/errors.js:1143:9)\n    at asyncHandler (/opt/hostedtoolcache/node/22.17.0/x64/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/protocol.js:487:57)\n\nAlso teardown failed:\nSeveral failures occurred:\n\n1) CalledProcessError: Command '['adb', 'pull', '/data/local/tmp/Beta_3_-_Test_record_video.mp4', '/home/runner/work/test-uiautomator2-driver/test-uiautomator2-driver/TestRecord/Beta_3_-_Test_record_video.mp4']' returned non-zero exit status 1.\n\n2) File '/home/runner/work/test-uiautomator2-driver/test-uiautomator2-driver/TestRecord/Beta_3_-_Test_record_video.mp4' does not exist.\n\n3) FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/test-uiautomator2-driver/test-uiautomator2-driver/TestRecord//Beta_3_-_Test_record_video.mp4'","time":{"start":1750911974453,"stop":1750911980430,"duration":5977}}]},"d9d7a72455dc6fe7bf78c2dfd37539b3":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"3b6048a41d5aeedb","status":"failed","statusDetails":"Setup failed:\nWebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: Error executing adbExec. Original error: 'Command '/usr/local/lib/android/sdk/platform-tools/adb -P 5037 -s emulator-5554 shell 'settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy'' exited with code 1'; Command output: adb: device offline\n\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: Error executing adbExec. Original error: 'Command '/usr/local/lib/android/sdk/platform-tools/adb -P 5037 -s emulator-5554 shell 'settings delete global hidden_api_policy_pre_p_apps;settings delete global hidden_api_policy_p_apps;settings delete global hidden_api_policy'' exited with code 1'; Command output: adb: device offline\n\n    at getResponseForW3CError (/opt/hostedtoolcache/node/22.17.0/x64/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/errors.js:1143:9)\n    at asyncHandler (/opt/hostedtoolcache/node/22.17.0/x64/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/protocol.js:487:57)\n\nAlso teardown failed:\nSeveral failures occurred:\n\n1) CalledProcessError: Command '['adb', 'pull', '/data/local/tmp/Beta_2_-_Test_send_message.mp4', '/home/runner/work/test-uiautomator2-driver/test-uiautomator2-driver/TestRecord/Beta_2_-_Test_send_message.mp4']' returned non-zero exit status 1.\n\n2) File '/home/runner/work/test-uiautomator2-driver/test-uiautomator2-driver/TestRecord/Beta_2_-_Test_send_message.mp4' does not exist.\n\n3) FileNotFoundError: [Errno 2] No such file or directory: '/home/runner/work/test-uiautomator2-driver/test-uiautomator2-driver/TestRecord//Beta_2_-_Test_send_message.mp4'","time":{"start":1750911968333,"stop":1750911974443,"duration":6110}}]},"9b89755166e3b0dcbf1eb84eb702700a":{"statistic":{"failed":1,"broken":0,"skipped":0,"passed":0,"unknown":0,"total":1},"items":[{"uid":"aba8666827596020","status":"failed","statusDetails":"Setup failed:\nWebDriverException: Message: An unknown server-side error occurred while processing the command. Original error: The instrumentation process cannot be initialized within 30000ms timeout. Make sure the application under test does not crash and investigate the logcat output. You could also try to increase the value of 'uiautomator2ServerLaunchTimeout' capability\nStacktrace:\nUnknownError: An unknown server-side error occurred while processing the command. Original error: The instrumentation process cannot be initialized within 30000ms timeout. Make sure the application under test does not crash and investigate the logcat output. You could also try to increase the value of 'uiautomator2ServerLaunchTimeout' capability\n    at getResponseForW3CError (/opt/hostedtoolcache/node/22.17.0/x64/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/errors.js:1143:9)\n    at asyncHandler (/opt/hostedtoolcache/node/22.17.0/x64/lib/node_modules/appium/node_modules/@appium/base-driver/lib/protocol/protocol.js:487:57)","time":{"start":1750911994308,"stop":1750912034978,"duration":40670}}]},"bbe0e327cb132618698cd2daa90e8228":{"statistic":{"failed":0,"broken":0,"skipped":1,"passed":0,"unknown":0,"total":1},"items":[{"uid":"9dbed9cdfb5e3ed4","status":"skipped","statusDetails":"15 != 14","time":{"start":1750911980441,"stop":1750911994296,"duration":13855}}]}}